dependencies.yaml

```diff
@@ -251,6 +251,7 @@ dependencies:
           - sphinx-markdown-tables
           - sphinx<6
           - sphinxcontrib-websupport
+          - pylibcugraphops=23.6.*
```

Comment:
can you add an entry for this file in `update-version.sh`?

Reply:
update and in latest push